GoogleForCreators/web-stories-wp
# Code Quality: Clean up shareable components between checklist and help center

## Context
Part of cleanup after ppcv2

## Summary
Just cleans up some of the dependencies between the ppcv2 popup and the help center popup so the ppcv2 component isn't dependent on the help center

## Relevant Technical Choices
NA


## To-do
NA


## User-facing changes
NA


## Testing Instructions
Open and play with the help center, checklist & shortcuts menu in the bottom right corner of the editor and see that no behavior has changed.


### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8122
